.github/workflows/unit-tests.yml

```diff
@@ -10,12 +10,12 @@ jobs:
       fail-fast: false # do not fail fast, let all the failing tests fail.
       matrix:
         php: [8.0]
-        es_version: [7.17.5]
+        es_version: [7.17.5, 8.10.2]
         multisite: [0]
         wp_version: ["latest"]
         include:
           - php: '7.4'
-            es_version: '7.17.5'
+            es_version: '8.10.2'
```

Comment:
🤷. Testing matrixes are hard to balance. I don't think we have to worry too much about the interaction of ES version, PHP version, WP version, and multisite vs single-site (otherwise, we'll end up with 18 or even 36 jobs). As long as we're running at least one job in each bucket, I expect we'll catch anything that comes up.

As to if we should deprecate 6.8, meh, I'm fine keeping it for now.

I take some pride that SP would still work all the way down to the 0.x beta releases of ES, as impractical as that is. At some point, we could probably clean things up by removing support for ES <7, but that's a task for another day.